Content.Shared/Chemistry/Solution/Components/RefillableSolutionComponent.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Robust.Shared.GameObjects;
+using Robust.Shared.Serialization.Manager.Attributes;
+using Robust.Shared.ViewVariables;
+
+namespace Content.Shared.Chemistry.Solution.Components
+{
+    [RegisterComponent]
+    public class RefillableSolutionComponent : Component
+    {
+        public override string Name => "RefillableSolution";
+
+        [ViewVariables(VVAccess.ReadWrite)]
+        [DataField("solution")]
+        public string Solution { get; set; } = "default";
```

Comment:
As elsewhere please remove default solutions and just use default! If they don't specify it should cause an exception.

Reply:
Some tests like to add components. And when they add components, they don't initialize them. So Solution is not `""` or a sensible default but null, which explodes the test. I asked to create a bunch of Init methods, but some Maintainer or above told me to just leave a default value.